src/common.c

```diff
@@ -1822,7 +1823,7 @@ static void parse_logging_options(int opt_type, char *opt_str)
 
 		for (i = 0; rtr_logging_groups[i] != NULL; i++) {
 			if (rtr_check_config_token(rtr_logging_groups[i], opt_str, sep, &reverse)) {
-				int bit_val = (i == 0) ? 0x01 : i * 2;
+				int bit_val = exp2(i);
```

Comment:
I'm pretty sure `1 << i` does what you want here without needing libm and floating point.